src/nvim/screen.c

```diff
@@ -4511,25 +4571,11 @@ void screen_adjust_grid(ScreenGrid **grid, int *row_off, int *col_off)
 // @param count max number of signs
 // @param[out] n_extrap number of characters from pp_extra to display
 // @param[in, out] sign_idxp Index of the displayed sign
-static void get_sign_display_info(
-    bool nrcol,
-    win_T *wp,
-    sign_attrs_T sattrs[],
-    int row,
-    int startrow,
-    int filler_lines,
-    int filler_todo,
-    int count,
-    int *c_extrap,
-    int *c_finalp,
-    char_u *extra,
-    size_t extra_size,
-    char_u **pp_extra,
-    int *n_extrap,
-    int *char_attrp,
-    int *draw_statep,
-    int *sign_idxp
-)
+static void get_sign_display_info(bool nrcol, win_T *wp, sign_attrs_T sattrs[], int row,
+                                  int startrow, int filler_lines, int filler_todo, int count,
+                                  int *c_extrap, int *c_finalp, char_u *extra, size_t extra_size,
```

Comment:
should we take the opportunity to switch away from this "align with (" style? so this would become:

```
static void get_sign_display_info(bool nrcol, win_T *wp, sign_attrs_T sattrs[], int row,
    int startrow, int filler_lines, int filler_todo, int count, int *c_extrap,
    int *c_finalp, char_u *extra, size_t extra_size, char_u **pp_extra,
```

Reply:
IMHO no. Extremely long argument argument lists are pretty rare (and an antipattern, this should be refactored to use a struct later on) and our existing style looks good for the common case (_not_ extremely long arg lists)